main.go

```diff
@@ -71,17 +118,30 @@ func main() {
 		log.Infof("Uploading file %s", cpuFile)
 		cpuResponse, err := uploadFile(envVars.UploadAPIURL, envVars.Token, cpuValues)
 		if err != nil {
-			log.WithError(err).Errorf("Failed to upload file %s", cpuFile)
+			return errors.Errorf("Failed to upload file %s", cpuFile)
```

Comment:
better to wrap the error `errors.Wrapf(err, "Failed to upload file %s", cpuFile)